stories/atoms/components/list/List.stories.mdx

```diff
@@ -0,0 +1,276 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { List } from 'ui/atoms/list/List'
+import { ListItem } from 'ui/atoms/listItem/ListItem'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { ProgressBar } from 'ui/atoms/progressBar/ProgressBar'
+import { Icon } from 'ui/atoms/icon/Icon'
+import { Button } from 'ui/atoms/button/Button'
+import { Toast } from 'ui/atoms/toast/Toast'
+import { useState } from 'react'
+
+<Meta
+  title="Atoms/List"
+  component={List}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code'
+      }
+    }
+  }}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="component-story-main">
+          <ThemeSwitcher />
+          <div
+            style={{
+              display: 'grid',
+              gap: '25px'
+            }}
+          >
+            <Story />
+          </div>
+        </div>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# List
+
+> A primary UI component to display a list of items.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Description
+
+The `List` allows to display a list of elements and manage them.
+
+## Overview
+
+<Story
+  name="Overview"
+  argTypes={{
+    items: { control: false }
```

Comment:
Could you enhance the `items` type in the story?
The arguments' table shows `ListItemProps[]` as type but I'm not sure this is understandable for the user..

Reply:
Ok. Is it ok to put `ListItem` which is defined right after?

Reply:
I think we should expose the real type, so the user does not have to switch on `ListItem` story to look for it.
```suggestion
    items: { 
      control: false, 
      table: {
        type: { 
          summary: 'Array<{title?: string, description?: string | JSX.Element, leftElement?: JSX.Element, rightElement?: JSX.Element}>' }
      } }
```